requester/requester.go

```diff
@@ -154,3 +149,18 @@ func (r *Requester) appendRecord(rec Record) {
 		r.numErr++
 	}
 }
+
+// helpers
+
+// Request.Clone() method from net/http packacke does not clone the body, so it is added after with .GetBody()
```

Comment:
nitpick, but kinda related to idiomatic/conventional go:

Unexported methods are helpers, there's no real need to point that out. Also if a function is documented, the documentation explains what the function does :

```go
// cloneRequest fully clones a http.Request by also cloning the body via Request.GetBody.
func cloneRequest(req *http.Request) (*http.Request, error)
```

Rule of thumb is function documentation should be written thinking about how it will appear in godocs (command `make docs`).

Reply:
Thanks! Duly noted :)